action.yml

```diff
@@ -105,6 +109,30 @@ runs:
             mkdir -p doc
             cp -r "$TMP_DIR/doc/htm" doc/
           fi
+
+      - name: "Copy extra files from the release"
+        shell: bash 
+        working-directory: gh-pages
+        if: ${{ ! inputs.extra-files == '' }}
+        run: |
+          for PAIR in $(echo "${{ inputs.extra-files }}" | tr "," "\n"); do
+
+            SRC=$(echo  ${PAIR} | cut -d':' -f1)
+            DEST=$(echo ${PAIR} | cut -d':' -f2)
```

Comment:
Perhaps if `PAIR` contains no `:` we could set `SRC=$PAIR` and `DST=$PAIR` so that we don't have to write e.g. `tst:tst` ?

Reply:
(ultra super duper nitpick, please ignore: if you name it DST instead of DEST it has the same width as `SRC` ;-). But using DEST is fine, too, of course)